astro/src/content/docs/apis/_apikey-get-response-body.mdx

```diff
@@ -5,6 +5,9 @@ import JSON from 'src/components/JSON.astro';
 #### Response Body
 
 <APIBlock>
+  <APIField name="apiKey.expirationInstant" type="Long">
+    The expiration [instant](/docs/reference/data-types#instants) of this API key. Once an API key has expired, it will no longer be accepted for [API Authentication](/docs/apis/authentication).
```

Comment:
I think that you should explicitly mention that the key will 401 if expired. I could see people being confused that their key is "bad" when it is just "expired".

Reply:
If you are mentioning that here, I'd also add a note to the troubleshooting section here: https://fusionauth.io/docs/apis/authentication#troubleshooting 

and to the api key authentication section here: https://fusionauth.io/docs/apis/authentication#api-key-authentication